### DISCUSSION TITLE
Syntax abreviada de propriedades CSS considerado um anti-pattern

### DISCUSSION THREAD

**Discussion by leandroruel** [frontendbr/forum#390] [Dicas]
[li este artigo](http://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/) interessante do mago do css, e criador do ITCSS harry roberts, que fala sobre os problemas que podem ser causados ao usar formas abreviadas (shorthands) de propriedades CSS, dêem uma lida e comentem o que acham:
http://csswizardry.com/2016/12/css-shorthand-syntax-considered-an-anti-pattern/

**Comment by hjdesigner** [comment]
Legal esse modo de pensar, uso bastante abreviado e nunca tinha parado para pensar que estamos estilizando valores que não vamos usar.

**Comment by worldvini** [comment]
Sim, é interessante, eu tinha começado a pensar sobre isso, quando comecei a utilizar o flex-box, pois com o shorthands, várias configurações default quebram todo o layout.
Então após isso eu comecei a pensar no que realmente vale a pena usar o shorthands ou pegar somente a propriedade especifica que vou utilizar/estilizar.
